doc/source/Manual.rst

```diff
@@ -1859,7 +1859,7 @@ If you have Git_ installed on your development system,
 you can use it together with pip
```

Comment:
Git is no longer required, when using  `pip install https://github..`

May be this whole paragraph need a bit of a change:
- If one wants to fix a bug, she is strongly recommended to use git and issue a pull-request at github. So installing without git is only for those who don't have git or dont't want to use it.
- If one only wants to install the latest version, she does not need git when using  `pip install https://github..`